frontend/components/SignUpForm.js

```diff
@@ -1,14 +1,26 @@
 import React from 'react'
+import superagent from 'superagent'
 import { Button, Form, Header, Segment, Input, Message } from 'semantic-ui-react'
 
 import ExternalAuthProviders from './ExternalAuthProviders'
 import useForm from '../hooks/useForm'
 import { SignUpForm } from '../models/SignUpForm'
 
+const endpoint = `${process.env.KITSPACE_GITEA_URL}/user/kitspace/sign_up`
+
+
 export default function () {
   const [form, onChange, isValid, errors] = useForm(SignUpForm)
 
-  const submit = () => {}
+  const submit = async () => {
```

Comment:
@kasbah  
- It result in cors error.
- Access to XMLHttpRequest at 'http://gitea.kitspace.test:3000/user/kitspace/sign_up' from origin 'http://kitspace.test:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
- Aren't the Nginx configuration suppose to solve this? I know I am missing something.

Reply:
It hasn't been enabled for this case / endpoint I believe. We should be able to solve it with either nginx config or within gitea.

Reply:
Basically we have to allow the apex domain  origin (`kitspace.test`) on these endpoints somehow.